dockerHashing/src/test/scala/cromwell/docker/DockerImageIdentifierSpec.scala

```diff
@@ -17,6 +17,7 @@ class DockerImageIdentifierSpec extends AnyFlatSpec with CromwellTimeoutSpec wit
       ("index.docker.io/ubuntu",         Option("index.docker.io"),   None,                   "ubuntu",     "latest"),
       ("broad/cromwell/submarine",                None,               Option("broad/cromwell"), "submarine",  "latest"),
       ("gcr.io/google/slim",              Option("gcr.io"),         Option("google"),         "slim",     "latest"),
+      ("us-central1-docker.pkg.dev/google/slim",              Option("us-central1-docker.pkg.dev"),         Option("google"),         "slim",     "latest"),
```

Comment:
ToL: The original intent of horizontal whitespace alignment seems to have been abandoned [as was foretold](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment). Would you consider removing the extra &emsp;&emsp;&emsp;&emsp; spaces in &emsp; this &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;block of &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp; code?